README.md

```diff
@@ -11,24 +11,52 @@ This repository contains the Coq development of the paper:
 
 ### Prerequisites ###
 
-This development requires Coq v8.7.1 to work, as well as the following libraries:
-- [Mathematical Components](https://math-comp.github.io/math-comp/) 1.6.4
-- [Extensional Structures](https://github.com/arthuraa/extructures) 0.1
-- [CoqUtils](https://github.com/arthuraa/coq-utils/releases/tag/v0.1) 0.1
+This development requires Coq v8.9.1 to work, as well as the following libraries:
+- [Mathematical Components](https://math-comp.github.io/math-comp/) 1.9.0 or dev / on branch master
+- [CoqUtils](https://github.com/arthuraa/coq-utils) on branch master (tested with commit [54c1269](https://github.com/arthuraa/coq-utils/commit/54c1269e1e85e14404e9dab3805e9db448f419f0)),
```

Comment:
The order of dependencies is wrong here: void and deriving are needed by CoqUtils, so they should appear before.

Reply:
Also, what's "dev /" supposed to mean? I've never heard that before.

Reply:
> The order of dependencies is wrong here: void and deriving and extstructures are all needed by CoqUtils, so they should appear before it.

Yes indded, since I let opam handle the dependencies I put this one first which fetches all the other ones.

> Also, what's "dev /" supposed to mean? I've never heard that before.

This kind of version is activated if you add the `extra-dev` repo (with some dev versions specific to coq versions, for instance for quickchick, 8.9.dev tracks the 8.9 branch of their repo)

Reply:
Changed this bit, silenced some polluting warnings about extraction and with extracted programs when using OCaml >= 4.08.0